mantis-common/src/main/java/com/mantisrx/common/utils/JsonUtility.java

```diff
@@ -16,15 +16,14 @@
 
 package com.mantisrx.common.utils;
 
+import mantis.com.fasterxml.jackson.core.type.TypeReference;
```

Comment:
We probably don't want to commandeer the `mantis` namespace globally. We could relocate packages into `io.mantisrx.shaded`.

Reply:
Makes sense. I was following the pattern from mantis-rxnetty. So we may need to relocate that if we decide to merge it into this shaded project in the future.

Reply:
Ah, yeah I agree `io.mantisrx.shaded` is the place to put it.